homeassistant/data_entry_flow.py

```diff
@@ -81,7 +82,7 @@ class FlowResult(TypedDict, total=False):
     context: dict[str, Any]
     result: Any
     last_step: bool | None
-    options: Mapping[str, Any]
+    menu_options: list[str] | Mapping[str, Any]
```

Comment:
It was not. That is for another step.

```suggestion
    options: Mapping[str, Any]
    menu_options: list[str] | Mapping[str, Any]
```